p2p/__init__.py

```diff
@@ -602,44 +602,49 @@ def clone_content_item(self, slug, clone_slug, keep_embeds=False, keep_relateds=
                 }
                 payload['related_items'].append(related_item)
 
+        found_contributors = find_clone_contributors(content_item)
+
+        if 'contributors' in content_item:
```

Comment:
(irrelevant to james' concern) if that's the case, shouldn't the method name be `clone_story_or_htmlstory_content_item` then?